build-tools-internal/src/main/groovy/elasticsearch.build-scan.gradle

```diff
@@ -99,7 +99,28 @@ buildScan {
         link 'Source', "https://github.com/elastic/elasticsearch/tree/${BuildParams.gitRevision}"
       }
     } else {
-      tag 'LOCAL'
+
+      URL buildKiteUrl = System.getenv('BUILDKITE_BUILD_URL') ? new URL(System.getenv('BUILDKITE_BUILD_URL')) : null
```

Comment:
Since we have the build url can we add that as a "CI Build` link like we do for Jenkins?

Reply:
done